wrf.rb

```diff
@@ -41,6 +43,8 @@ def install
       # which is not installed. Therefore, we build it here and install it in the WPS subfolder
       # of WRF.
       wps_args = std_cmake_args + %W[
+        -DCMAKE_C_COMPILER=gcc
+        -DCMAKE_Fortran_COMPILER=gfortran
```

Comment:
Again, gcc won't resolve to the GCC you want it to (you'll still get apple's clang) and there's no need for the Fortran explicit dependency, it's the only option.

```suggestion
```